resources/prepare_aws_package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Do you need to include the APK in this zipfile? How do you pass it to AWS Device Farm?

Reply:
Application Under Test(AUT) or the apk will be uploaded separately through boto3 command.